10_fibonacci/README.md

```diff
@@ -2,7 +2,9 @@
 
 Create a function that returns a specific member of the Fibonacci sequence:
 
-> A series of numbers in which each number ( Fibonacci number ) is the sum of the two preceding numbers. The simplest is the series 0, 1, 1, 2, 3, 5, 8, etc.
+> A series of numbers in which each number ( Fibonacci number ) is the sum of the two preceding numbers.
+> In this exercise, the Fibonacci sequence used is 1, 1, 2, 3, 5, 8...
```

Comment:
```suggestion
> In this exercise, the Fibonacci sequence used is 1, 1, 2, 3, 5, 8, etc.
```

I added back the `etc.` for I think it describes the continuance of the sequence better than `...` 
What do you think?

If you agree, feel free to `commit suggestion` with the button so you don't have to locally rewrite and push :)